frontend/document/koptinterface.lua

```diff
@@ -36,16 +36,41 @@ end
 
 function ContextCacheItem:dump(filename)
     if self.kctx:isPreCache() == 0 then
-        logger.dbg("dumping koptcontext to", filename)
-        return serial.dump(self.size, KOPTContext.totable(self.kctx), filename)
+        logger.dbg("Dumping KOPTContext to", filename)
```

Comment:
For my own knowledge: what is stored in thie KOPTContext ? and why it matter it is cached?

Reply:
Not a clue :D. But a lot of stuff for k2pdfopt, apparently.